SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -705,6 +713,13 @@ - (SDLSecondaryTransportType)sdl_getTransportTypeFromProtocol:(SDLProtocol *)pro
     }
 }
 
+- (void)appDidBecomeReady {
+    self.secondaryProtocol.securityManager = self.primaryProtocol.securityManager;
+    if ([self.stateMachine.currentState isEqualToString:SDLSecondaryTransportStateConfigured]) {
+         [self.stateMachine transitionToState:SDLSecondaryTransportStateConnecting];
+    }
```

Comment:
Do you not need to set the security manager here?

Reply:
removed

Reply:
I don't think you understood my original comment. I think you _do_ need to set it here, right? Or not? If not, why not?